appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/annotation/handlers/AbstractAuthAnnotationHandler.java

```diff
@@ -256,18 +255,16 @@ private Set<MethodDescriptor> getMethodAllDescriptors(
     private boolean hasMethodPermissionsFromDD(MethodDescriptor methodDesc,
             EjbDescriptor ejbDesc) {
         Map methodPermissionsFromDD = ejbDesc.getMethodPermissionsFromDD();
-        if (methodPermissionsFromDD != null) {
-            Set allMethods = ejbDesc.getMethodDescriptors();
-            for (Object mdObjsObj : methodPermissionsFromDD.values()) {
-                List mdObjs = (List)mdObjsObj;
-                for (Object mdObj : mdObjs) {
-                    MethodDescriptor md = (MethodDescriptor)mdObj;
-                    for (Object style3MdObj :
-                            md.doStyleConversion(ejbDesc, allMethods)) {
-                        MethodDescriptor style3Md = (MethodDescriptor)style3MdObj;
-                        if (methodDesc.equals(style3Md)) {
-                            return true;
-                        }
+        Set allMethods = ejbDesc.getMethodDescriptors();
+        for (Object mdObjsObj : methodPermissionsFromDD.values()) {
+            List mdObjs = (List) mdObjsObj;
+            for (Object mdObj : mdObjs) {
+                MethodDescriptor md = (MethodDescriptor) mdObj;
+                for (Object style3MdObj :
```

Comment:
I would prefer for structure on the same line for legibility:
```suggestion
                for (Object style3MdObj : md.doStyleConversion(ejbDesc, allMethods)) {
```